services/ranks_service.py

```diff
@@ -71,9 +71,19 @@ async def get_is_private(self, user_id: str) -> bool:
         is_private = await self.db_conn.fetchval('SELECT is_private FROM ranks '
                                                  'WHERE user_id = $1',
                                                  user_id)
```

Comment:
```suggestion
                                                 user_id)
        return is_private
```
It seems you've accidentally deleted the return statement here